MultigridProjector/Extensions/MyObjectBuilderExtensions.cs

```diff
@@ -71,5 +76,96 @@ public static MyObjectBuilder_Toolbar GetToolbar(this MyObjectBuilder_TerminalBl
 
             return null;
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static string FormatBlockName(MyObjectBuilder_TerminalBlock terminalBlockBuilder)
+        {
+            if (terminalBlockBuilder.CustomName != null)
+                return terminalBlockBuilder.CustomName;
+
+            var defaultName = MyDefinitionManager.Static?.TryGetDefinition<MyCubeBlockDefinition>(terminalBlockBuilder.SubtypeId, out var blockDefinition) == true ? blockDefinition.DisplayNameText : terminalBlockBuilder.SubtypeId.ToString();
```

Comment:
throws this:
```
[PluginLoader] [Error] CS0165: Use of unassigned local variable 'blockDefinition' in file:
multigrid-projector-ff9143a627450e6162f4bf9dd4a74794ec3ec085/MultigridProjector/Extensions/MyObjectBuilderExtensions.cs (86,166)
```
any chance 'blockDefinition' should be '**B**lockDefinition' ?

Reply:
It makes no difference.
(And yes, I tested this just for you)